examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -93,6 +96,111 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
     );
   }
 
+  Widget _buildAlarmMenu() {
+    final String time = _alarmHour.toString().padLeft(2, '0') + ': ' +
```

Comment:
Don't reinvent this. Use DateTime or Duration.

Reply:
I have tried but DateTime and Duration don't have a nice way of showing the clock as string.
For example, Duration will show 12:03 as "12:03:00.000000", which does not look really nice.
Anyway, after I implement the builder() for CupertinoPicker, I will update this demo in Gallery to show current date, so I can update this issue later.

Reply:
The general way of formatting a date in Dart is to use the DateFormatter in package:intl/intl.dart.

@xster do we have a formatter utility that doesn't depend on package:intl?